tests/test_conversation_manager.py

```
@@ -0,0 +1,558 @@
+        assert topic == "Bayern Munich"
+
+    def test_feed_offer_not_repeated(self):
+        """Test that feed offer is only made once."""
+        manager = ConversationManager()
+
+        # Create pattern
+        manager.add_turn("Bayern game?", "Saturday")
+        manager.add_turn("Bayern score?", "3-1")
+
+        # First check - should offer
+        should_offer1, topic1 = manager.should_offer_feed()
```

Ruff (0.14.2)

Unpacked variable `topic1` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)